src/code/circuit_python/index.md

```diff
@@ -0,0 +1,19 @@
+# Circuit Python
+
+[Circuit Python](https://docs.circuitpython.org/) is a dialect of the
+Python programming language targetting microcontrollers. It is maintained
+by [Adafruit](https://adafruit.com).
+
+The programming experience with Circuit Python is based on writing and
+copying Python source files to the device through your computer's file
+system. This means you can write Circuit Python code in any text editor.
+With that said, there is an ecosystem of recommended tools that make the
+process even easier.
+
+Documentation for the Circuit Python language and core libraries are
+available at
+[https://docs.circuitpython.org/](https://docs.circuitpython.org/).
```

Comment:
When creating a link who's target is itself, use the condensed syntax:

`<https://docs.circuitpython.org>`